github/doc.go

```diff
@@ -85,9 +85,11 @@ Rate Limiting
 
 GitHub imposes a rate limit on all API clients. Unauthenticated clients are
 limited to 60 requests per hour, while authenticated clients can make up to
-5,000 requests per hour. To receive the higher rate limit when making calls
-that are not issued on behalf of a user, use the
-UnauthenticatedRateLimitedTransport.
+5,000 requests per hour. The Search API has a custom rate limit. Authenticated
+clients can make up to 30 requests per minute. Unauthenticated clients are
```

Comment:
It seems a bit odd that the previous sentence describes unauthenticated first, authenticated second, but this one inverts that order.

Also, you could join these two sentences with "... per minute, **while** authenticated clients...", just like above. Optional and up to you.